src/Auth.ts

```diff
@@ -260,22 +302,6 @@ export class Auth {
     return response.accessToken;
   }
 
-  private async getClientApplication(logger: Logger, debug: boolean): Promise<Msal.ClientApplication | undefined> {
```

Comment:
This method was only used twice, and in one of those case I've excluded all application auth types because that made more sense. Using just getPublicClient and getConfidentialClient started to made more sense then.

```ts
if (this.connection.authType !== AuthType.Certificate &&
      this.connection.authType !== AuthType.Secret &&
      this.connection.authType !== AuthType.Identity) {
      this.clientApplication = await this.getPublicClient(logger, debug);
//...
}
```